src/fs-utils.js

```diff
@@ -0,0 +1,41 @@
+import { copy } from 'fs-extra';
+import { Promise } from 'bluebird';
+import temp from 'temp';
+import fs from 'fs';
+
+const log = require('debug')('electron-windows-installer:fs-utils');
+
+temp.track();
+
+const promisedCopy = Promise.promisify(copy);
+const promisedCreateTempDir = Promise.promisify(temp.mkdir);
+const readFileAsync = Promise.promisify(fs.readFile);
+const readDirAsync = Promise.promisify(fs.readdir);
+const unlinkAsync = Promise.promisify(fs.unlink);
+const writeFileAsync = Promise.promisify(fs.writeFile);
+const renameAsync = Promise.promisify(fs.rename);
+const inspectAsync = Promise.promisify(fs.stat);
+
+export {
```

Comment:
Please consider to get rid of `Async` suffix. We don't need it — async method just return Promise. Node doesn't use `Async` suffix.